src/layouts/components/Editable/AddSectionButton.tsx

```diff
@@ -0,0 +1,54 @@
+import { HStack, Icon, MenuItemProps, Text } from "@chakra-ui/react"
+import { Menu, MenuButtonProps } from "@opengovsg/design-system-react"
+import { BiPlus } from "react-icons/bi"
+
+export const AddSectionButton = ({ children, ...rest }: MenuButtonProps) => {
+  return (
+    <Menu isStretch>
+      {({ isOpen }) => (
+        <>
+          <Menu.Button
+            variant="outline"
+            isStretch
+            isOpen={isOpen}
+            rightIcon={undefined}
+            // NOTE: Need `sx` here as there is styling set on the exported `Menu.Button` component
+            // that overrides the `textAlign` prop
+            sx={{
```

Comment:
wait so our figma deviates from ds? is this intended?

Reply:
good catch! i added this in cos it wasn't aligning (if memory serves, but this was ages ago). i tried removing and nothing seemed to changed so... removed!